po/nl.po

```diff
@@ -586,200 +700,240 @@ msgstr "LMS-formaat"
 
-#: src/widgets/preferences_window.rs:481
+#: src/widgets/preferences_window.rs:482
 msgid "HWB"
 msgstr "HWB"
 
-#: src/widgets/preferences_window.rs:485
+#: src/widgets/preferences_window.rs:486
 msgid "CIELCh / HCL"
 msgstr "CIELCh/HCL"
 
-#: src/widgets/preferences_window.rs:491
+#: src/widgets/preferences_window.rs:492
 msgid "Name"
 msgstr "Naam"
 
-#: src/widgets/preferences_window.rs:494 src/window.rs:397 src/window.rs:420
-#: src/window.rs:580 src/window.rs:651
+#: src/widgets/preferences_window.rs:495 src/window.rs:432 src/window.rs:455
+#: src/window.rs:630 src/window.rs:701
 msgctxt "Information that no name for the color could be found"
 msgid "Not named"
 msgstr "Geen naam beschikbaar"
 
-#: src/widgets/preferences_window.rs:499
+#: src/widgets/preferences_window.rs:500
 msgid "LMS"
 msgstr "LMS"
 
-#: src/widgets/preferences_window.rs:503
+#: src/widgets/preferences_window.rs:504
 msgid "Hunter Lab"
 msgstr "Hunter-Lab"
 
 #. Translators: This should not be translate, Please enter your credits here instead (format: "Name https://example.com" or "Name <email@example.com>", no quotes)
-#: src/widgets/about_window.rs:50
+#: src/widgets/about_window.rs:51
 msgid "translator-credits"
 msgstr "Philip Goto <philip.goto@gmail.com>"
 
-#: src/widgets/about_window.rs:110
+#: src/widgets/about_window.rs:111
 msgctxt ""
 "The changelog of the current version. Should be similar to the CHANGELOG.md "
 "file at the project root. Please ensure that the tags have a matching "
 "closing tag. Since xgettext does not recognize rust multiline strings, this "
 "should be on a single line using line breaks (\\n) for new lines."
 msgid ""
-"<p>A new release with exciting new features.</p><p>New Features:</p>\n"
-"<ul><li>Customize formats in the settings</li><li>Export the colorscheme as "
-"a GIMP palette using the new export option</li><li>A placeholder is shown "
-"when no color is picked</li><li> Hunter-Lab and LMS color spaces have been "
-"added</li><li>The GNOME color palette has been added</li><li>Fixed minor "
-"issues</li></ul>"
+"<p>A minor release with many small QoL improvements.</p><p>Added:</p>\n"
+"<ul><li>Show a toast with an Undo option when clearing the history</"
+"li><li>Use Ctrl-Q to close the window</li><li>Use a default minimum window "
+"size, content should no longer unnecessarily cut off</li><li>New icon by "
+"@bertob</li><li>Export palettes as Adobe Swatch Exchange (ASE), hex, PAL, "
+"Paint.NET (txt) files</li><li>Minor changes to the store metadata</li></"
+"ul><p>Changed:</p>\n"
+"<ul><li>Hunter-Lab now displays 0 instead of NaN</li><li>Improved tooltips "
+"for copying colors</li><li>Palette window now uses a button with a more "
+"explicit text instead of an icon</li><li>Reworded project description</"
+"li><li>Updated all screenshots</li></ul><p>Fixed:</p>\n"
+"<ul><li>Separator is now hidden when the history list is hidden</"
+"li><li>Improved editing experience when change the Hex color/name</li></ul>"
 msgstr ""
```

Comment:
I think you missed one here

Reply:
Ah, I'm sorry. I might have done it a little too quickly. I'll try to finish that last one soon!

Reply:
Done!